Source/Upstream: Yes, in v238-stable branch

From 0f61b3b0ded362b81f79b541545c46e49133f4bb Mon Sep 17 00:00:00 2001
From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Wed, 14 Mar 2018 14:52:45 +0900
Subject: [PATCH 40/51] man: mention 'status' is the default command for
 localectl

(cherry picked from commit d4e0773b4da6146f64947360ae6f8c971d459372)
---
 man/localectl.xml | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/man/localectl.xml b/man/localectl.xml
index 89a7ab285..eff8bc3c9 100644
--- a/man/localectl.xml
+++ b/man/localectl.xml
@@ -118,6 +118,10 @@
       <xi:include href="standard-options.xml" xpointer="version" />
       <xi:include href="standard-options.xml" xpointer="no-pager" />
     </variablelist>
+  </refsect1>
+
+  <refsect1>
+    <title>Commands</title>
 
     <para>The following commands are understood:</para>
 
@@ -125,8 +129,8 @@
       <varlistentry>
         <term><command>status</command></term>
 
-        <listitem><para>Show current settings of the system locale and
-        keyboard mapping.</para></listitem>
+        <listitem><para>Show current settings of the system locale and keyboard mapping.
+        If no command is specified, this is the implied default.</para></listitem>
       </varlistentry>
 
       <varlistentry>
-- 
2.16.3

